sources/packages/backend/package.json

```diff
@@ -68,6 +68,7 @@
     "passport-jwt": "^4.0.1",
     "pg": "^8.5.1",
     "qs": "^6.9.6",
+    "redis": "^4.6.6",
```

Comment:
mm.. I was thinking that we already had redis as some dependencies

Reply:
No, we never had redis(client lib) used before, as we use nestjs bull to connect to redis. We did have the ioredis library for cluster connection. For typeorm cache we need the redis library.

![image](https://user-images.githubusercontent.com/54600590/235808348-b95c19fb-7487-4f4f-b0ab-6051b7ab6c1f.png)